catalog/app/containers/Bucket/PackageDialog/FilesInput.tsx

```diff
@@ -1671,182 +1812,3 @@ export function FilesInput({
     </Root>
   )
 }
-
-const useFilesSelectorStyles = M.makeStyles((t) => ({
-  checkbox: {
-    color: `${t.palette.action.active} !important`,
-    padding: 3,
-    '&:hover': {
-      backgroundColor: `${fade(
-        t.palette.action.active,
-        t.palette.action.hoverOpacity,
-      )} !important`,
-    },
-    '& svg': {
-      fontSize: '18px',
-    },
-  },
-}))
-
-interface FilesSelectorEntry {
-  type: 'dir' | 'file'
-  name: string
-  selected: boolean
-  size?: number
-  meta?: Types.JsonRecord
-}
-
-export type FilesSelectorState = FilesSelectorEntry[]
-
-interface FilesSelectorProps {
-  input: {
-    value: FilesSelectorState
-    onChange: (value: FilesSelectorState) => void
-  }
-  className?: string
-  disabled?: boolean
-  errors?: Record<string, React.ReactNode>
-  meta: {
-    submitting: boolean
-    submitSucceeded: boolean
-    submitFailed: boolean
-    dirty: boolean
-    error?: string
-    submitError?: string
-    initial: FilesSelectorState
-  }
-  title: React.ReactNode
-  truncated?: boolean
-}
-
-export function FilesSelector({
```

Comment:
could you please move this change to a separate PR to reduce the size of this one?